src/textual/app.py

```diff
@@ -529,6 +532,25 @@ def return_value(self) -> ReturnType | None:
         """
         return self._return_value
 
+    @property
+    def return_code(self) -> int:
```

Comment:
The `return_code` property will be 1 when no error has occurred. That would be astonishing if you inspect it before the app exits.

I know you said it was "meaningless" but I don't think we should have meaningless properties... 

I would expect either the return code to be `0` or `None` if it hasn't been set.

Reply:
Fair point.
As of da0cd5e, the return code is `None` before/while running the app.
(So as to avoid confusion from a successful exit with return code `0`.)